sdk/spring/spring-messaging-azure-storage-queue/src/main/java/com/azure/spring/messaging/storage/queue/core/properties/StorageQueueProperties.java

```diff
@@ -52,6 +52,10 @@ public String getConnectionString() {
         return connectionString;
     }
 
+    /**
+     * Set the connection string to connect to the service.
```

Comment:
better add an empty line here
<img width="693" alt="image" src="https://user-images.githubusercontent.com/4465723/157361658-cdbf56ff-96c2-486c-ad92-f2a7491b9af0.png">

Reply:
Hey, I noticed that most of the setters' javadoc in our xxProperties.java don't have that empty line, need we follow that?

Reply:
either is okay

Reply:
it's up to you @yiliuTo 

FYI: [example](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#examples)

Reply:
We have a lot of things to do to improve the java doc.

6 month ago, I create an issue: #24332, which is intended to do this. But this issue is closed without satisfy the requirement.

Since we are in hurry to release 4.0, we do not have enough time to improve the java doc. My current suggestions is improve the java doc's quality for new classes and new public methods.

We can get a lot of documents by searching by google. For example: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html